src/main/kotlin/com/beust/kobalt/plugin/application/ApplicationPlugin.kt

```diff
@@ -29,8 +29,8 @@ class ApplicationConfig {
 }
 
 @Directive
-fun Project.application(init: ApplicationConfig.() -> Unit) {
-    ApplicationConfig().let { config ->
+fun Project.application(init: ApplicationConfig.() -> Unit): ApplicationConfig {
```

Comment:
Trying to decide if it makes sense to return an `ApplicationConfig` from `application{}`...